performance_search_max_throughput_test.py

```diff
@@ -142,7 +142,10 @@ def run_search_best_performance(self, stress_cmd_tmpl: str,  # pylint: disable=t
                         break
                 else:
                     self.log.info("Decrease number of loaders and back to start num of process")
-                    self.loaders.nodes = original_nodes_list[:len(self.loaders.nodes) - num_loaders_step]
+                    new_num_of_nodes = len(self.loaders.nodes) - num_loaders_step
+                    if new_num_of_nodes < 1:
+                        break
```

Comment:
Probably a log message may be useful here.

Reply:
Added.